wip Nfultz/next

nfultz opened the pull request.

ben-oneill committed:
Add files via upload

ben-oneill committed:
Update conf.R

Fix error in CONF.mean (previously used sample variance instead of sample deviation for interval)

ben-oneill committed:
Add files via upload

nfultz commented:
Just a heads up, `pmatch` is a base R function. Its fine to collide with it, but if there's an alternative name for this distribution you were considering, it's one more point for it.

nfultz committed:
Fix test for bugfix in CONF.mean

nfultz committed:
Add missing Suggest

nfultz committed:
Add missing imports for matching distribution

nfultz committed:
Fixing documentation

nfultz committed:
removing redudant file

ben-oneill commented:
Thanks for spotting that.  In that case, perhaps change it to matching (i.e., dmatching, pmatching, qmatching, rmatching, etc.).


________________________________
From: Neal Fultz ***@***.***>
Sent: Wednesday, 17 November 2021 6:48 AM
To: ben-oneill/stat.extend ***@***.***>
Cc: Subscribed ***@***.***>
Subject: Re: [ben-oneill/stat.extend] wip Nfultz/next (PR #1)


Just a heads up, pmatch is a base R function. Its fine to collide with it, but if there's an alternative name for this distribution you were considering, it's one more point for it.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://apac01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fben-oneill%2Fstat.extend%2Fpull%2F1%23issuecomment-970623793&data=04%7C01%7C%7Cb90ea7b7db724c5ac19c08d9a93a18e5%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637726889274256063%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=nIIg7F3XYL5GCwDhH9K0YphKasU%2FzRjM3PxNp8M7aIM%3D&reserved=0>, or unsubscribe<https://apac01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FALMWGCUIPKJJIC36UUI2Y4LUMKYRXANCNFSM5IFCVRRQ&data=04%7C01%7C%7Cb90ea7b7db724c5ac19c08d9a93a18e5%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637726889274266022%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=VSvMXIw4UyVXq%2BQ8QpWZkEz9CW4FFc9KGsJZyMBGR5Y%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://apac01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7C%7Cb90ea7b7db724c5ac19c08d9a93a18e5%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637726889274266022%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=9IcD1D2Wap3vUdY%2FbyYOD6Vof%2BOAqas7Z3EmaF9rdMg%3D&reserved=0> or Android<https://apac01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7C%7Cb90ea7b7db724c5ac19c08d9a93a18e5%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637726889274275976%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=DTzAeRRk4fNleaGgk8hIAhQDgqdNU%2F80QNfnQWRNhe0%3D&reserved=0>.

nfultz committed:
Finalizing lsubfactorial

nfultz committed:
Renaming match to matching

nfultz committed:
Matching functions documentation.

nfultz committed:
add lsubfactorial example

ben-oneill committed:
Update HDR.R

Changed name in table from HDR.match to HDR.matching

ben-oneill committed:
Update and rename dmatch.R to dmatching.R

Altered internal code for computing matching distribution (more efficient)

ben-oneill committed:
Update and rename moments.match.R to moments.matching.R

Changed name from match to matching

ben-oneill committed:
Update HDR.R

Updated internal code to compute matching distribution in HDR.matching (more efficient)

ben-oneill committed:
Add files via upload

nfultz committed:
Fix NAMESPACE for rename

ben-oneill committed:
Update MLE.matching.R

nfultz committed:
Fixing namespace for MLE.matching

nfultz committed:
docs for MLE.matching

nfultz committed:
increase version

ben-oneill committed:
Major update of dmatching and others

Generalised the distribution to add a 'trials' parameter and allow approximation using the normal distribution

ben-oneill committed:
Update HDR.matching

Added an additional parameter 'trials' and added option to use normal approximation

ben-oneill committed:
Update moments.matching.R

Added additional parameter 'trials'

ben-oneill committed:
Add files via upload

nfultz committed:
Merge pull request #2 from ben-oneill/ben-more-changes

Ben more changes

nfultz commented on R/HDR.R:
```diff
@@ -579,3 +580,152 @@ print.hdr <- function(x, ...) {
+      
+      #Deal with case where n = Inf and prob = 0
+      if ((n == Inf)&(prob == 0)) {
+        LOGPROBS <- dpois(x, lambda = 1, log = TRUE) }
```
@ben-oneill 
R CMD Check says that `x` is not in scope here. This was probably copy-pasted from somewhere else, and you missed an edit. Same issue in `qmatching()`

For HDR, if its asymptotically poission, could just return the result of HDR.poission. Same strategy for qmatching.

nfultz committed:
check fixes

nfultz committed:
Fix broken example

nfultz committed:
Fix pmatching signature.

ben-oneill committed:
Update HDR.R

Fixed error in function

ben-oneill committed:
Update dmatching.R

Fixed errors in the probability functions

nfultz committed:
roxygenize and missing import

nfultz commented:
I pulled in your other changes and fixed the warnings; haven't checked the outputs specifically though. I sent a build to rhub and winbuilder, assumming those are ok you can submit to cran.
